### DISCUSSION TITLE
Strange_insults.html uses some words inappropriate for a school setting

### DISCUSSION THREAD

**Discussion by dpandey2021** [phcode-dev/phoenix#1930] [General]
Under the example HTML Games, strange_insults.html has some inappropriate language for a school setting. 

However, i do understand the code itself does have educational value. Can the file be modified or removed here? 

It does not violate any of the rules in the source project: https://github.com/Metroxe/one-html-page-challenge

**Comment by abose** [comment]
Thanks for bringing this to our notice. We will remove it from Phoenix code in the next release. The html challenge repo is not controlled by us, but we will remove the file in the code distributed inside Phoenix Code.

To get it removed from the html challenge repo itself, can you raise an issue there too.

**Comment by abose** [comment]
This has been fixed as part of https://github.com/phcode-dev/phoenix/pull/1931
The file will be auto deleted on the next update of phoenix code. Thanks again for reporting!

Since a lot of students visit the discussions page for help, i am closing the discussion for reducing visibility of this issue.
Please feel free to reopen the issue or comment on it for any help.
